eleventy.config.cjs

```diff
@@ -137,6 +139,59 @@ module.exports = function (
   );
 
   // For making a non-nested fallback
+
+  // PurgeCSS filter to extract only used CSS
+  eleventyConfig.addFilter(
+    "purgeCSS",
+    async (
+      /** @type {string} */ css,
+      contentPaths = [
+        "./pages/**/*.html",
+        "./src/_includes/**/*.html",
+        "./src/js/**/*.mjs"
```

Comment:
Add "./src/css/**/*.css"

Reply:
Fixed it with adding State template JS file 
```"./node_modules/@cagovweb/state-template/dist/js/cagov.core.min.js"```